internal/cli/alerts/settings_fields_type.go

```diff
@@ -32,13 +32,16 @@ func (opts *ConfigFieldsTypeOpts) init() error {
 	return err
 }
 
+var matcherFieldsTemplate = `FIELD_TYPE:{{range .}} {{.}} {{end}}
```

Comment:
I don't field like we need a title here, if you think it adds any value please make it just `FIELD`